### DISCUSSION TITLE
Importing HtmlEscapedString

### DISCUSSION THREAD

**Discussion by laurentpellegrino** [honojs/hono#2374] [Q&A] [answered]
Hey there!

Just discovered honojs and looking to build a small app on top of it. While testing, I needed to import `HtmlEscapedString` in Typescript but I cannot find a way. It looks like the type is not exported. Is there a reason?

**Comment by yusukebe** [comment, marked as answer]
Hi @lpellegr 

Can you import it from `hono/utils/html`?

**Comment by laurentpellegrino** [comment]
@yusukebe Hey, thanks. Indeed, this works.
